Zapper-fi/studio
# feat: Add a Balance fetcher

## Description
Add a balance fetcher for Thales, which includes the following parts:
- stakedBalanceOf (StakingThales),  
- totalAccountEscrowedAmount(EscrowThales), 
- balanceOf (THALES/ETH LP Staking) 

## Checklist

- [X] (optional) As a contributor, my Ethereum address/ENS is: leifu.eth

## How to test?
1. Pick some users randomly and call the API to check the balance
e.g. `http://localhost:5001/apps/thales/balances?addresses[]=[user wallet]&network=optimism`


## Screenshots
http://localhost:5001/apps/thales/balances?addresses[]=0x3ebe9f00c3fccd09294a053dfe4ccf5a70787082&network=optimism
![image](https://user-images.githubusercontent.com/4819006/168182892-80b8d379-0c37-40d8-b14f-8638aa230891.png)
